Makefile

```diff
@@ -519,9 +519,14 @@ electron-develop:
 		-t electron \
 		-s "$(PLATFORM)"
 
+babel:
+	$(NPX) babel --copy-files --out-dir lib lib && $(NPX) babel --copy-files --out-dir tests tests
```

Comment:
So, from what I can tell from a quick skim through this PR, this will convert e.g. `lib/gui/components/safe-webview.jsx` into `lib/gui/components/safe-webview.js` ? Should we also be checking these 'output' files into Git, just like we do for the 'output' sass files? (and if so, I guess we'll also want a `ensure-staged-babel.sh` CI script?) Or if these files should remain outside of git, I guess they should be added to `.gitignore`?

Reply:
@jviotti what do you think about checking them into git vs not doing that?

Reply:
I think with this change you should really be compiling the etcher UI into a build directory. Keeping source and compiled files side-by-side is not good.
Using webpack to compile the code into a bundle would be my preferred approach.

Reply:
Yeah, having them together will make it difficult to only ship the built ones in the asar.

Reply:
So bundle it is? Sounds good to me

Reply:
> having them together will make it difficult to only ship the built ones in the asar.

I guess the same thing applies to the css and scss fles?